core/include/traccc/edm/ccl_partition.hpp

```diff
@@ -0,0 +1,33 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+#pragma once
+
+// traccc include(s).
+#include "traccc/edm/container.hpp"
+
+namespace traccc {
+
+namespace partitioning {
+// Define number of cells to put together in each partition. Equal to the number
+// of threads per block in the CCL kernel.
+static constexpr std::size_t MAX_CELLS_PER_PARTITION = 1024;
+}  // namespace partitioning
+
+/// CCL partition used for dividing cell collection into similarly sized
+/// fragments
+struct ccl_partition {
```

Comment:
Sounds good to me, I might even generalise it a bit more and make it `traccc::partition` . As I suspect it might come in handy in the future for other types other cells.